addons/l10n_it_edi_ndd/models/account_move.py

```diff
@@ -22,12 +22,16 @@ class AccountMove(models.Model):
     @api.depends('line_ids.matching_number', 'payment_state')
     def _compute_l10n_it_payment_method(self):
```

Comment:
```suggestion
    def _compute_l10n_it_payment_method(self):
        if self.env.company.account_fiscal_country_id.code != 'IT':
            return
```
This would prevent calling `search` on `account.move.line` for non Italian companies.